src/metadataGeneration/methodGenerator.ts

```diff
@@ -38,7 +38,7 @@ export class MethodGenerator {
     responses.push(this.getMethodSuccessResponse(type));
 
     return {
-      deprecated: isExistJSDocTag(this.node, tag => tag.tagName.text === 'deprecated'),
```

Comment:
You can't remove the functionality that exists for users to already mark a method as deprecated via JSDocs. If you want this PR to be approved, you'll have to have `this.getIsDeprecated` check for the presence of a `@Deprecated` decorator or a JSDoc deprecated tag.

Reply:
You are right. I just modified `this.getIsDeprecated` to consider JSDoc deprecated tag